src/Microsoft.Azure.WebJobs.Extensions.Storage/WebJobs.Extensions.Storage.csproj

```diff
@@ -96,12 +96,10 @@
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.7" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.1.7" />
     <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="11.1.7" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.22" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
intentionally removing project reference and relying in nuget?

Reply:
Yes, this repo has already moved forward to 3.0.23 -- so any build we generate now will need to wait (or i'll need to roll that back, which is confusing). This let's us release "out-of-band" by directly targeting 3.0.22, which is already out in nuget.